routes/web.php

```diff
@@ -1,18 +1,13 @@
 <?php
+use App\Http\Controllers\LanguageController;
 
+use App\Http\Controllers\TaskController;
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::resource('/',TaskController::class);
+Route::get('language/{language}',[LanguageController::class,'index'])->name('language');
+Route::resource('/task', TaskController::class)->only([
+    'destroy'
+]);
+Route::post('/task', [TaskController::class,'store']);
```

Comment:
thiếu dòng trắng cuối file
```suggestion
Route::post('/task', [TaskController::class, 'store']);

```